### ISSUE TITLE
2.0.0-beta3-DSM6  - confusing for configuration 

### ISSUE THREAD

**Issue by erkobg** [openhab/openhab-syno-spk#27]
Hello,
introducing OpenHab2 is a big step forward and I really appreciate your efforts.
I installed it and as already pointed it is working well.

Except it is confusing or impossible to configure.
There are several folders with configurations :
**/volume1/public/OpenHAB2**   - has conf, addon and configuration folder 

**/var/packages/OpenHAB2/target   = /volume1/@appstore/OpenHAB2**
 has conf, addon and configuration folder 

I tried to run demo sitemap - but no success.
I tried putting it in all the folders, delete some - result no demo sitemap.

I also tried adding HABMIN in all the addons folders but it was not running.

Any instruction or simplification will be appreciated.

**Comment by erkobg** [comment]
As always if you want to achieve something, do it yourself... 

For the history:
1.  before installing OpenHab2 uninstall and delete all traces from previous OpenHab
    delete /volume1/public/OpenHAB2
   delete /var/packages/OpenHAB2   =/volume1/@appstore/OpenHAB2
              
2. Install OpenHab2
3. You are ready 
4. if you want to start demo sitemap 
        navigate to` /volume1/@appstore/OpenHAB2/conf/services`
       then` nano addons.cfg`
       and then uncomment the line `**package=** `and set it to `**demo**`

Now you have demo running.


For more follow:
http://docs.openhab.org/tutorials/

**Comment by cniweb** [comment]
Thanks @erkobg. The `configurations` folder has been renamed to `conf`

**Comment by enricobaros** [comment]
Hi,

I just tested the installation and now it's correctly installed and started.
I suppose that the installation script needs to be adapted to create the link  
/volume1/@appstore/OpenHAB2/conf to /volume1/public/OpenHAB2/conf

Warning : when stopping the package from the center, it doesn't stop really OH, it's still running.

**Comment by cniweb** [comment]
Thanks @enricobaros the stop works in OH2 with stop.sh, I adapt this, too!

**Comment by cniweb** [comment]
Hi,

I add a new release, with conf folder and correct stopping the package:
https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3-DSM6
Try it, please and provide me feedback.

Thanks,
Chris

**Comment by enricobaros** [comment]
Hi Chris,

Thank you.

It’s not yet ok. 
In the file scripts <https://github.com/openhab/openhab-syno-spk/tree/2.0-DSM-6.0/scripts>/installer.sh,
you need to chagnge the line 91also :     rm -r ${SYNOPKG_PKGDEST}/configurations

Also, the stop from the synoloy interface does not yet the stop.

Best regards,
Eric

**Comment by erkobg** [comment]
Hello,
thanks for the latest release,
but it seems that it is not starting after install - I mean by the interface.

Regards

**Comment by cniweb** [comment]
Hi @enricobaros,

Thanks for your feedback, I add a new release, with corrections:
https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3-DSM6
Try it, I can´t test (I have no DiskStation with DSM6) and provide me feedback, please.

Thanks,
Chris

**Comment by enricobaros** [comment]
Hi Chris,

Thank for this update, the links are now correctly done on /conf and /addons folder in the public main folder. (Just don’t forget to adapt the description shown in the console where « configurations » is still given.

But now it is no more starting by the normal way, I don’t even see the /logs directly being created.

Best regards,
Eric

**Comment by dirtydevious** [comment]
I've installed 2.0.0-beta3-DSM6 and it does not start the webpage at http://x.x.x.x8080/start/index any help would be greatly appreciated.

**Comment by aurelio1** [comment]
Hi cniweb
I installed 2.0.0-beta3 but I found that if I want to participate in testing 2.0.0 I would need nightly builds. What are the differences between your syno-package and the released version? Is it just a "one-button-installer" that puts the package into @appstore and are the other documents the same as I can download from the main sources? And did you take your source from the snapshots on [http://www.openhab.org/getting-started/downloads.html](http://www.openhab.org/getting-started/downloads.html)? Thank you for your help.
